README.md

```diff
@@ -124,6 +124,32 @@ The list of available actions can be seen via `medic-conf --help`.
 * upload JSON files as docs on instance
 * compress PNGs and SVGs in the current directory and its subdirectories
 
+## Editing contacts across the hierarchy. 
+To edit existing couchdb documents, create a CSV file that contains the id's of the document you wish to update, and the columns of the document attribute(s) you wish to add/edit. By default, values are parsed as strings. To parse a CSV column as a JSON type, refer to the [Property Types](#property-types) section to see how you can parse the values to different types. Also refer to the [Excluded Columns](#excluded-columns) section to see how to exclude column(s) from being added to the docs.
+
+Parameter | Description | Required 
+-- | -- | --
+column(s) | Comma delimited list of columns you wish to add/edit. If this is not specified all columns will be added. | No
+docDirectoryPath | This action outputs files to local disk at this destination | No. Default `json-docs`
```

Comment:
You should also document the new argument `file`/`files`

Reply:
Done.